SYCL/ESIMD/api/functional/ctors/ctor_default.cpp

```diff
@@ -20,22 +20,20 @@
 // created (https://github.com/intel/llvm/issues/5077) and the TEST_HALF macros
 // must be enabled when it is resolved.
 
-#include "../common.hpp"
+#include "common.hpp"
 
 using namespace cl::sycl;
 using namespace sycl::ext::intel::experimental::esimd;
+using namespace esimd_test::api::functional::ctors;
 using namespace esimd_test::api::functional;
```

Comment:
```suggestion
```
Same for the `ctor_copy.cpp`

Reply:
When I apply this comment I got the following compilation error:
```
ctor_default.cpp:122:39: error: use of undeclared identifier 'tested_types'
  const auto types = get_tested_types<tested_types::all>();
ctor_default.cpp:123:21: error: use of undeclared identifier 'get_all_dimensions'; did you mean 'esimd_test::api::functional::get_all_dimensions'?
  const auto dims = get_all_dimensions();
                    ^~~~~~~~~~~~~~~~~~
                    esimd_test::api::functional::get_all_dimensions
SYCL\ESIMD\api\functional\ctors/../type_coverage.hpp:96:6: note: 'esimd_test::api::functional::get_all_dimensions' declared here
auto get_all_dimensions() { return values_pack<1, 8, 16, 32>(); }
```